static/templates/message_edit_form.handlebars

```diff
@@ -1,6 +1,6 @@
 {{! Client-side Mustache template for rendering the message edit form. }}
 
-<form id="message_edit_form" class="form-horizontal">
+<form id="message_edit_form">
```

Comment:
As far as i can see Bootstrap 2 supports spacing when stacking columns for smaller devices. In this case it seems like its better to use form with the default styles (  without any required base class on the `<form>` ) @rishig